.gitignore

```diff
@@ -50,3 +50,6 @@ modules.order
 Module.symvers
 Mkfile.old
 dkms.conf
+
+#IntelliJ CLion Files
+.idea/
```

Comment:
You can set up a global gitignore for all your local clones with this global gitignore being local to your computer this way:

```sh
git config --global core.excludesfile ~/some/path/to/your/global/gitignore
```

This way you won't have to add lines specific to your own setup on any project you work on. =)